tweepy/error.py

```diff
@@ -30,4 +31,23 @@ class RateLimitError(TweepError):
     """Exception for Tweepy hitting the rate limit."""
     # RateLimitError has the exact same properties and inner workings
     # as TweepError for backwards compatibility reasons.
-    pass
+
+    def reset_on(self):
```

Comment:
Is this naming meant to match something else in the codebase? If not, I might call it something like `resets_in_sec` or `resets_in` since the "on" implies you're returning a timestamp (and the reset will happen on that timestamp) vs. some more like a countdown

Reply:
it doesn't need to match something else in the code, addressed in c1c0139